lib/bindings/HTTPBindings.js

```diff
@@ -57,6 +57,7 @@ function handleError(error, req, res, next) {
         name: error.name,
```

Comment:
I think the PR is almost ready (good work! :). However, you should include an entry in the CHANGES_NEXT_RELEASE file in describing the changes. Maybe something like the following:

```
- Hardening; software quality improvement based on ISO25010 recommendations
```

Reply:
fixed in 484e833100ab2fee19a4e8ecaf07628e43286918